src/pages/costModels/costModel/costCalc.styles.ts

```diff
@@ -1,12 +1,13 @@
+import global_FontSize_md from '@patternfly/react-tokens/dist/js/global_FontSize_md';
 import global_FontSize_xl from '@patternfly/react-tokens/dist/js/global_FontSize_xl';
-import global_spacer_lg from '@patternfly/react-tokens/dist/js/global_spacer_lg';
 import React from 'react';
 
 export const styles = {
   card: {
-    minHeight: '130px',
-    marginLeft: global_spacer_lg.value,
-    marginRight: global_spacer_lg.value,
+    minHeight: '150px',
```

Comment:
This `minHeight` value doesn't work with a string value. When defining styles like this, the value should just be a number `150`.

Reply:
done